serverlessworkflow/modules/ROOT/pages/eventing/consume-produce-events-with-knative-eventing.adoc

```diff
@@ -152,17 +152,17 @@ The following table lists the configuration property related to Knative sink gen
 
 |`org.kie.kogito.addons.knative.broker`
 |`default`
-|This property indicates the name of the default Knative broker that is deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative triggers, which are responsible to delegate the events that the Kogito service consumes.
+|This property indicates the name of the default Knative broker that is deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative triggers, which are responsible to delegate the events that the service consumes.
```

Comment:
```suggestion
|This property indicates the name of the default Knative broker that is deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative triggers, which are responsible to delegate the events that the workflow service consumes.
```
It is more obvious which service we talk about this way imho.

Reply:
Done